src/box/sql/vdbe.c

```diff
@@ -1265,7 +1265,7 @@ case OP_FunctionByName: {
 	uint32_t size;
 	struct Mem *mem = (struct Mem *)port_get_vdbemem(&ret, &size);
 	if (mem != NULL && size > 0)
-		*pOut = mem[0];
+		mem_copy(pOut, &mem[0]);
```

Comment:
Where is `mem` destroyed if you copy it instead of moving? The mem object itself is on the region, but its content could be on the heap, couldn't it?

Reply:
True, I should be using mem_move() instead of mem_copy() here. Fixed.